src/utils/api/helper.ts

```diff
@@ -698,6 +698,7 @@ export const proceedToPayment = async (
         apiPaymentEcommerceClientWithRetry.requestTransactionAuthorization({
           bearerAuth,
           transactionId,
+          lang: localStorage.getItem("i18nextLng") ?? window.navigator.language,
```

Comment:
the `window.navigator.language` return a string in the 'it-IT' format that differ from the `i18nextLng` language format that is in 2 char format. 
Change this to set `it` as default language in case of missing local storage property

Reply:
![image](https://github.com/user-attachments/assets/966f0053-2c36-40dc-8731-c7005bd185ec)
apparently this changes on safari, because on edge i get "it"
we can definitelly split this on "-" to get the wanted result